Small grammar changes

andynava opened the pull request:
Reorder settings to match settings on the screen
Clarify where the credentials will be used
Remove "v" after screenshot
Change case of "Q" to match UI
Add commas to indicate there are several steps to view streaming log

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
Small grammar changes

Reorder settings to match settings on the screen
Clarify where the credentials will be used
Remove "v" after screenshot
Change case of "Q" to match UI
Add commas to indicate there are several steps to view streaming log
